contracts/src/teleporter/registry/TeleporterUpgradeable.sol

```diff
@@ -35,18 +35,26 @@ abstract contract TeleporterUpgradeable is
 {
     using SafeERC20 for IERC20;
 
-    // The Teleporter registry contract manages different Teleporter contract versions.
-    TeleporterRegistry public teleporterRegistry;
-    /**
-     * @dev A mapping that keeps track of paused Teleporter addresses.
-     */
-    mapping(address teleporterAddress => bool paused) private _pausedTeleporterAddresses;
+    // solhint-disable private-vars-leading-underscore
+    /// @custom:storage-location erc7201:teleporter.storage.TeleporterUpgradeable
```

Comment:
For my own understanding, do you know why the custom storage location has three forward slashes? This seems to be the convention in OZ's docs as well as the EIP 7201 spec, but I couldn't find any justification for it.

Reply:
Not understanding the rationale behind this was bothering me also...dug into it and my mind was blown 😅 
![image](https://github.com/user-attachments/assets/0c84e04d-9550-43e1-a491-f7eb659a138d)

Reply:
So "///" is for single line natspec comments, and "/**" is for multi-line natspec comments.

(Where all natspec comments start with a @ tag, and can be used to auto-generate docs)